### PR TITLE
Changes needed for PyPi Module

### PR DESCRIPTION
**Pull request by agt24**
Added pyproject.toml
Moved some yaml files and removed older versions
Change name of repo to conform with other extensions in the NWB catalog
Updates to README.md so links will still work when published at pypi.org
Updates to README.md to prepared for the migration to the histedlab org as the main location

### CODE REVIEW

File: src/pynwb/ndx_photostim/spec/ndx-photostim.namespace.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+namespaces:
+- author:
+  - Carl Harris
```
Comment (histed) [inline src/pynwb/ndx_photostim/spec/ndx-photostim.namespace.yaml:4]: @agt24 @lafosse I think we said we'd add Paul here as co-author due to his contributions, and add him as an email contact too, yes?

I would just change this but I'm not sure if it needs to be done anywhere else.

**Review by histed — CHANGES_REQUESTED** [review]
All looks great save the authorship comment I made.

Going to ass Paul as a reviewer - I think that is the right thing instead of tagging him, but not 100% sure.

File: src/pynwb/ndx_photostim/spec/ndx-photostim.namespace.yaml (hunk 1 above)
Comment (agt24) [reply src/pynwb/ndx_photostim/spec/ndx-photostim.namespace.yaml:4]: Yes, definitely. This yaml file was previously in a different directory, I just moved it to the spec subdirectory. Hadn't notice it contained an author field, but I've now added Paul's name and email.

**Comment by agt24** [comment]
@histed  thanks for the comment about the potential need to fix the author list in other places. I searched the repo for Carl's name and found several other files. All of them now list both Carl and Paul.

File: src/pynwb/ndx_photostim/spec/ndx-photostim.namespace.yaml (hunk 1 above)
Comment (lafosse) [reply src/pynwb/ndx_photostim/spec/ndx-photostim.namespace.yaml:4]: This and the other additions all look good to me. Thanks!

**Pull request merged**